libcontainer/configs/validate/validator.go

```diff
@@ -128,6 +128,44 @@ func (v *ConfigValidator) cgroupnamespace(config *configs.Config) error {
 	return nil
 }
 
+// convertSysctlVariableToDotsSeparator can return sysctl variables in dots separator format.
+// The '/' separator is also accepted in place of a '.'.
+// Convert the sysctl variables to dots separator format for validation.
+// More info:
+//   https://man7.org/linux/man-pages/man8/sysctl.8.html
+//   https://man7.org/linux/man-pages/man5/sysctl.d.5.html
+// For example:
+// Input sysctl variable "net/ipv4/conf/eno2.100.rp_filter"
+// will return the converted value "net.ipv4.conf.eno2/100.rp_filter"
+func convertSysctlVariableToDotsSeparator(val string) string {
+	if val == "" {
+		return val
+	}
+	sepFunc := func(r rune) bool {
+		return r == '.' || r == '/'
+	}
+	firsSepIndex := strings.IndexFunc(val, sepFunc)
+	if firsSepIndex != -1 {
```

Comment:
This can be rewritten to simplify things a bit. Less indentation, avoiding else block etc.

For example:

```go
if firstSepIndex == -1 || val[idx] == '.' {
	return val
}
// Replace . with / and vice versa.
f := func(r rune) rune {
.....
return strings.Map(f, val)
```

Reply:
Thanks for advice. Updated.